cmd/hz/protobuf/ast.go

```diff
@@ -112,10 +116,22 @@ func astToService(ast *descriptorpb.FileDescriptorProto, resolver *Resolver) ([]
 			Name: s.GetName(),
 		}
 
+		baseDomain := ""
+		domainAnno := checkFirstOption(api.E_BaseDomain, s.GetOptions())
+		if args.CmdType == meta.CmdClient {
+			val, ok := domainAnno.(string)
+			if !ok || len(val) == 0 {
+				baseDomain = ""
+			} else {
+				baseDomain = val
+			}
+			service.BaseDomain = baseDomain
```

Comment:
122-128 lines are equal to:
```
service.BaseDomain = ""
val, ok := domainAnno.(string)
if ok && len(val) != 0 {
    service.BaseDomain = val
}
```
no need `baseDomain` && extra `else` clause?

Reply:
done